src/components/MultiHopTrade/components/MultiHopTradeConfirm/hooks/helpers.ts

```diff
@@ -23,48 +18,6 @@ export type GetAllowanceArgs = {
 export enum GetAllowanceErr {
```

Comment:
maybe better to live alongside the react query now?

Reply:
Might as well, this domain definitely needs some types module - 6e97f8247d

Reply:
Yeah agreed, there are a handful of domains that could use some common type/logic consolidation.